src/util/PasswordComplexityUtil.js

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) 2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['okta'], function (Okta) {
+
+  var _ = Okta._;
+
+  // return true if the complexity is a number and the value is greater than zero
+  var isComplexityEnabledPosNum = function(complexityValue) {
```

Comment:
I think at some point, we may have a rule that is enabled when the value is string or something else. And we want to reuse the function. So I named this function specific for a positive number.